src/subcommands/encrypt.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2022 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Encrypts data using a public key or the public part of a key pair.
+//!
+//! Will use the algorithm set to the key's policy during creation. Currently only
+//! supports asymmetric encryption such as RSA.
```

Comment:
This is a bit confusing - should the first line also mention ", or with a secret key"? Otherwise the first line already implies asymmetric encryption only.

Reply:
Yes, I had to re-read a couple of times, but then I got that it was conflating the semantics of the operation with its current functional status, after which it was clear how I could improve it!